com.unity.ml-agents/Tests/Editor/EditModeTestInternalBrainTensorGenerator.cs

```diff
@@ -63,8 +63,7 @@ static List<TestAgent> GetFakeAgents(ObservableAttributeOptions observableAttrib
             return agents;
         }
 
-        [Test]
-        public void Construction()
+        public void Construction(int apiVersion)
```

Comment:
What's going on here? It will no longer be called as a test, and doesn't look like it's called anywhere else (at least not in this file).

Reply:
You are right, this is unit test was not being used properly, I modified t back so it will run (There was another test in a similar situation that I also changed)